librz/core/cmd_analysis.c

```diff
@@ -8023,7 +8023,7 @@ static int cmd_analysis_all(RzCore *core, const char *input) {
 		}
 		break;
 	case '\0': // "aa"
-	case 'a': // "aa"
```

Comment:
The `if (*input == 'a')` in line 8058 seems to be redundant because its in `case 'a'` of `switch (*input)`, though it has nothing to do with the analysis improvement.